rhel/SPECS/burp.spec

```diff
@@ -23,12 +23,13 @@ backing up Windows computers.
 %setup -q -n %{name}-%{version}
 
 %build
+autoreconf -vif
```

Comment:
You'll need to BuildRequire autoconf, automake, libtool and pkg-config if you do this.